build/.goreleaser.yml

```diff
@@ -0,0 +1,76 @@
+builds:
+  - id: nri-oracledb-nix
+    main: ./src
+    binary: nri-oracledb
+    gobinary: "go1.11.13"
+    ldflags:
+      - -s -w -X main.integrationVersion={{.Version}} -X main.gitCommit={{.Commit}} -X main.buildDate={{.Date}}
+    goos:
+      - linux
+    goarch:
+      - amd64
+
+nfpms:
+  - id: nri-oracledb-deb
+    builds:
+      - nri-oracledb-nix
+    file_name_template: "{{ .ProjectName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure oracledb Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from oracledb components."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+    # Formats to be generated.
+    formats:
+      - deb
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+    files:
+      "oracledb-config.yml.sample": "/etc/newrelic-infra/integrations.d/oracledb-config.yml.sample"
+      "CHANGELOG.md": "/usr/share/doc/nri-oracledb/CHANGELOG.md"
+      "README.md":  "/usr/share/doc/nri-oracledb/README.md"
+      "LICENSE": "/usr/share/doc/nri-oracledb/LICENSE"
+    config_files:
+      "oracledb-definition.yml": "/var/db/newrelic-infra/newrelic-integrations/oracledb-definition.yml"
```

Comment:
It would be good to replace `files` and `config_files` with `content`, as in https://github.com/newrelic/nri-mssql/pull/48/commits/3e679b5a26b3f26f6bf77d3b6ccd68aa5db8a19d